cves/kernel/CVE-2016-3137.yml

```diff
@@ -328,8 +327,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: false
+  note: "No instances of developers vouching for any other person's work. No commit messages or anything in the bugzilla post referanced anything of that sort."
```

Comment:
typo in referenced, you have "referanced"

Reply:
fixed